services/online_exam_service/.env

```
@@ -36,10 +36,22 @@ RATE_LIMIT_MAX=100
 # BLOCKCHAIN CONFIG (Giữ nguyên các giá trị của bạn)
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
 GRADE_LEDGER_CONTRACT_ADDRESS=0xFf72998a0e9178Aa2890B82F90bAB2De0Cd118b0
-OWNER_ACCOUNT_PRIVATE_KEY=0x80801b1f0a74e06a6bc872b62b08ad7c9b17d4477f7d7cf5fe313f99573e7a2d
+OWNER_ACCOUNT_PRIVATE_KEY=0xaba9dede35ae3042e09949a82605a33f77b1602a6cc463ee7b829d663accf570
 
-# OTHER SERVICES URL
+# Services
 PROCTORING_SERVICE_URL=http://127.0.0.1:8082
+# Choose ONE of the token options below:
+# 1. Direct value (recommended for local dev)
+PROCTORING_SERVICE_TOKEN=Bearer mySecretKey12345678901234567890123456789012345678901234567890
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROCTORING_SERVICE_TOKEN key should go before the PROCTORING_SERVICE_URL key

(UnorderedKey)